transformer_engine/pytorch/tensor/float8_tensor.py

```diff
@@ -715,7 +715,7 @@ def __torch_dispatch__(cls, func, types, args, kwargs=None):
                 )
             # deep copy the scale inverse tensor and quantizer as well.
             scale_inv = tensor._scale_inv.detach().clone()
-            quantizer = tensor._quantizer.copy()
```

Comment:
I'm uncomfortable with this change since it means that an FP8 tensor created with `zeros_like` will share the same quantizer, and so could have strange non-local interactions. I assume that we are trying to avoid CPU overhead in `Quantizer.copy`, but do we understand exactly why that function is slow?